models/scanresults.go

```diff
@@ -351,6 +369,21 @@ func (r ScanResult) FormatExploitCveSummary() string {
 	return fmt.Sprintf("%d exploits", nExploitCve)
 }
 
+// FormatAlertsSummary returns a summary of XCERT alerts
```

Comment:
__[Golint]__ <sub>posted by Sider</sub>

comment on exported method ScanResult.FormatAlertSummary should be of the form "FormatAlertSummary ..."